.eslintrc.yaml

```diff
@@ -0,0 +1,71 @@
+# This is a baseline set of rules intended for usage in any ES6-based project.
+#
+# Be sure to add your project's specific environment(s) as needed via external configuration.
+#   See: http://eslint.org/docs/user-guide/configuring#specifying-environments
+
+---
+rules: # http://eslint.org/docs/rules
+
+  # Possible Errors
+
+  valid-jsdoc: [1]
+  curly: [2]
+  dot-notation: [2]
+  eqeqeq: [2]
+  no-div-regex: [2]
+  no-eval: [2]
+  no-extend-native: [2]
+  no-floating-decimal: [2]
+  no-lone-blocks: [2]
+  no-loop-func: [2]
+  no-multi-spaces: [2]
+  no-native-reassign: [2]
+  no-new-wrappers: [2]
+  no-return-assign: [2]
+  no-self-compare: [2]
+  radix: [2]
+  yoda: [2, "never"]
+
+  # Strict Mode
+
+  strict: [2, "never"]
+
+  # Variables
+
+  no-shadow: [2]
+  no-use-before-define: [2]
+
+  # Stylistic Issues
+
+  block-spacing: [2, "always"]
+  brace-style: [2, "1tbs", allowSingleLine: true]
+  camelcase: [2, properties: "always"]
+  comma-style: [2, "last"]
+  consistent-this: [2, "self"]
+  eol-last: [2]
+  indent: [2, 2, {"VariableDeclarator": { "var": 2, "let": 2, "const": 3}}]
+  linebreak-style: [2, "unix"]
+  newline-after-var: [2]
+  no-lonely-if: [2]
+  no-trailing-spaces: [2]
+  no-unneeded-ternary: [2]
+  quotes: [2, "backtick"]
```

Comment:
I vote against, because I want to believe our choices shape the future as much as the present shapes ours (reject the backticks, and work towards ripping it out of ES again), but will abide by whatever democratic decision we ends up concluding with.